src/components/dialog/dialog.tsx

```diff
@@ -129,12 +129,13 @@ export class Dialog {
         this.mdcDialog.listen('MDCDialog:closed', this.handleMdcClosed);
         this.mdcDialog.listen('MDCDialog:closing', this.handleMdcClosing);
 
-        this.mdcDialog.scrimClickAction = this.closingActions.scrimClick
-            ? 'close'
-            : '';
-        this.mdcDialog.escapeKeyAction = this.closingActions.escapeKey
-            ? 'close'
-            : '';
+        if (this.closingActions.scrimClick) {
+            this.mdcDialog.scrimClickAction = 'close';
+        }
+
+        if (this.closingActions.escapeKey) {
+            this.mdcDialog.escapeKeyAction = 'close';
+        }
```

Comment:
Have you tested these? My guess would be that scrimClick and escapeKey defaults to `close` and we would need to set them to `''` to prevent that behaviour?

Reply:
Yeah I tested it and it did not appear to have any effect. Even setting it to `''` did not seem to do anything. There might have been some change in Material that affected this behavior so we should probably look into it

Reply:
I take that back! :smile: